egograph/backend/dependencies.py

```
@@ -140,3 +147,23 @@ def get_google_health_daily_summary_use_case(
 ) -> GetGoogleHealthDailySummaryUseCase:
     """Google Health日次サマリ取得UseCaseを構築する。"""
     return GetGoogleHealthDailySummaryUseCase(repository)
+
+
+def get_timeline_repository(
+    config: BackendConfig = Depends(get_config),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)